File: samples/javascript_nodejs/05.multi-turn-prompt/userProfile.js
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

// @ts-check

class UserProfile {
    constructor(transport, name, age, picture) {
        this.transport = transport;
        this.name = name;
        this.age = age;
        this.picture = picture;
    }
}

module.exports.UserProfile = UserProfile;

File: archive/samples/java_springboot/05.multi-turn-prompt/src/main/java/com/microsoft/bot/sample/multiturnprompt/UserProfile.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

package com.microsoft.bot.sample.multiturnprompt;

import com.microsoft.bot.schema.Attachment;

/**
 * This is our application state.
 */
public class UserProfile {
    public String transport;
    public String name;
    public Integer age;
    public Attachment picture;
}
